tasks/20240522-fraxtal-root-gauge-factory/index.ts

```diff
@@ -0,0 +1,18 @@
+import { Task, TaskRunOptions } from '@src';
+import { FraxtalRootGaugeFactoryDeployment } from './input';
+
+export default async (task: Task, { force, from }: TaskRunOptions = {}): Promise<void> => {
+  const input = task.input() as FraxtalRootGaugeFactoryDeployment;
+
+  const args = [input.Vault, input.BalancerMinter, input.L1StandardBridge, input.FraxtalBAL, input.GasLimit];
+
+  const factory = await task.deployAndVerify('OptimismRootGaugeFactory', args, from, force);
```

Comment:
Should we call this (and the file) `FraxtalRootGaugeFactory` for consistency with other root gauges? Arbitrum also uses the same build-info, but is called "ArbitrumRootGauge". Even though it works, it looks like a "mistake" (e.g., in the readme, as we have two different tasks pointing to files of the same name).

Reply:
Arbitrum has its own gauge; this one has the exact same code as Optimism's gauge.
I tried doing that, but the problem is that the code inside the build info is called `OptimismRootGaugeFactory`. If I name it after Frax, it won't find the contract to deploy.

Reply:
To change the name we'd need to copy the file, rename the contract, and rebuild. I think it's not worth it.